roadmap.yaml

```diff
@@ -165,17 +167,29 @@ Data Engineering:
       - Amazon Kinesis
   Pipeline Orchestration:
     - Apache Airflow
+    - Prefect
+    - Dagster
     - Mage
     - Argo
   Messaging and Streaming Platform:
     - Apache Kafka
+    - Connector:
+        - Kafka Connect
+    - Schema Control:
+      - Schema Registry
```

Comment:
I am of the opinion that schema control is a distinct section. From a bird's-eye view perspective, it should be positioned as an independent section. It's not only related to the `messaging and streaming platform`.

Reply:
Let me separate it.

Reply:
Done

Reply:
Let's finalize this PR and have it in another PR.

Reply:
> Let's finalize this PR and have it in another PR.

Do you mean I make my change to another PR ?

Reply:
Hmm, I mean just remove the schema control from this PR and add it in another PR to discuss about it.

Reply:
Ahaa OK.

Reply:
Done.